main.go

```
@@ -0,0 +1,80 @@
+)
+
+func init() {
+	ReadFlag()
+	loadConfig()
+	setupLogger()
+}
+
+func main() {
+	flag.Parse()
+	database.SetDBInfo(cfg)
+	web.ListenAndServe(cfg)
```

golangci-lint (1.62.2)

Error return value of `web.ListenAndServe` is not checked

(errcheck)